agogosml_cli/Makefile

```diff
@@ -102,4 +105,10 @@ installedit: clean ## install the package while dynamically picking up changes t
 	pip install --editable .
 
 dockerbuild: clean
-	docker build -t agogosml_cli .
+	@if [ ! -z "${TOX_VERSION}" ]; then \
```

Comment:
Nitpick: `[ ! -z "${x}" ]` is equivalent to `[ -n "${x}" ]` or even just `[ "${x}" ]`.

From `man test`:

```
       -n STRING
              the length of STRING is nonzero

       STRING equivalent to -n STRING

       -z STRING
              the length of STRING is zero
```